lib/basicOracleDatabaseBuild.js

```diff
@@ -51,9 +51,9 @@ else {
 /**
 * Calling Git module to verify if both change/target branches are not out of sync.
 */
-async function _gitTargetSyncVerify(gitUrl, changeBranch, changeTarget, username, password) {
-const git = new Git();
-let result = await git.verify(changeBranch, changeTarget, username, password, gitUrl);
+async function _gitTargetSyncVerify() {
+const git = new Git(config.options.git);
+let result = await git.verify();
 if(result == 'True') {
   console.log("Successfully Verified that branch is not out of sync with target")
 }
```

Comment:
do we need to do if(result=='True') .? If we are expecting a boolean then we should only say if(result)